packages/features/ee/workflows/api/scheduleSMSReminders.ts

```diff
@@ -72,6 +73,11 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           ? reminder.booking?.attendees[0].timeZone
           : reminder.booking?.user?.timeZone;
 
+      const isAlphanumericSenderIdSupported = !noAlphanumericSenderIdSupport.find(
+        (code) => code === sendTo?.substring(0, code.length)
+      );
+      const senderID = isAlphanumericSenderIdSupported ? reminder.workflowStep.sender || "Cal" : "";
```

Comment:
```suggestion
      const senderID = isAlphanumericSenderIdSupported ? reminder.workflowStep.sender || "Cal.com" : "";
```
Can we use Cal.com as the sender ID instead? It's a little more clear than just Cal